### PR TITLE
ADX-394 OLD

### PR DESCRIPTION
**Pull request by Manoj-nathwani**
Moved to https://github.com/fjelltopp/ckanext-unaids/pull/68

### CODE REVIEW

**Review by tomeksabala — CHANGES_REQUESTED** [review]
You should start your branch from development not master.

File: ckanext/unaids/fanstatic/autocomplete-without-creating-new-options.js [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+ *            from the ajax returned results
+ * label    - A string of the label you want to appear within the dropdown for
+ *            returned results
+ *
+ * Examples
+ *
+ *   // <input name="tags" data-module="autocomplete" data-module-source="http://" />
+ *
+ */
+this.ckan.module('autocomplete-without-creating-new-options', function (jQuery) {
```
Comment (Manoj-nathwani) [inline ckanext/unaids/fanstatic/autocomplete-without-creating-new-options.js:18]: I changed this line

File: ckanext/unaids/fanstatic/autocomplete-without-creating-new-options.js [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+      }
+      else {
+        if (/MSIE (\d+\.\d+);/.test(navigator.userAgent)) {
+            var ieversion=new Number(RegExp.$1);
+            if (ieversion<=7) {return}
+         }
+      }
+
+      // disable creating new options
+      settings.createSearchChoice = false;
```
Comment (Manoj-nathwani) [inline ckanext/unaids/fanstatic/autocomplete-without-creating-new-options.js:75]: I changed this line

**Comment by Manoj-nathwani** [comment]
image

Fixed the `Add Member` page too ✅

**Comment by Manoj-nathwani** [comment]
Moved to https://github.com/fjelltopp/ckanext-unaids/pull/68

**Pull request closed without merging**